src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/CreateDescriptorsValidation.feature

```diff
@@ -1,10 +1,9 @@
-Feature: Descriptors are a set of mechanisms to support flexible enumerations or code sets. Validation of POST requests.
+Feature: Create a Descriptor
```

Comment:
This feature description is translated into the test names displayed in test results. The older long name is a good description when reading the feature file itself, but it turns out to be bad when trying to read test results. A short feature name is better.

Additional text [can be added](https://docs.reqnroll.net/latest/gherkin/feature-files.html) below the first line of the feature file, if we really need to add more detail for the reader.

![image](https://github.com/user-attachments/assets/5d38a495-5fba-4a4b-91e8-dfc6a2f99b10)